config/locales/en.yml

```diff
@@ -59,7 +59,7 @@ en:
     log_out: Log out
     my_dashboard: My Dashboard
     recent_activity: Recent Activity
-    report_problem: Report a problem
+    report_problem:
```

Comment:
Looks like this was inadvertant?

Reply:
Yes. I'm unsure how it happened, but I added it back.